scripts/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Christopher Jerdonek
+All rights reserved.
```

Comment:
We (Pew and Google) are sorting out the licensing issue for all of VIP. It will be an OSI-compliant license, so once we finalize that we will put a copy of that license in the main repo. Once we've done that, please update this file to point to the main LICENSE file.